DEVELOPMENT.md

```diff
@@ -275,6 +275,8 @@ kubectl apply -R -f config/monitoring/100-namespace.yaml \
 
 ## Iterating
 
+Before running the `./hack/update-codegen.sh` and `./hack/update-deps.sh` scripts, ensure that [mercurial](https://www.mercurial-scm.org/wiki/Download) is installed.
```

Comment:
Should we add this to the prerequisites above as well?

/ok-to-test

Reply:
yes, i think so too. Feel free to suggest a better comment about hg than what I included.